microservice-app/src/main/java/com/sdevelopment/skt/microservice/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.sdevelopment.skt.microservice.service.impl;
+
+import com.sdevelopment.skt.common.domain.Product;
+import com.sdevelopment.skt.microservice.service.ProductService;
+
+import java.util.List;
+
+public class ProductServiceImpl implements ProductService {
+
+    @Override
+    public void saveProduct(Product product) {
+
```

Comment:
please add proper documentation or comments indicating this logic will be implemented later

Reply:
Logic already implemented